src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -208,6 +222,10 @@ public String toString() {
             ", tcpNoDelay=" + this.tcpNoDelay +
             ", timeZone=" + this.timeZone +
             ", username='" + this.username + '\'' +
+            ", loadBalanceHosts=" + this.loadBalanceHosts +
+            ", hosts=" + hosts +
+            ", topologyKeys=" + this.topologyKeys +
+            ", YBServersRefreshInterval=" + this.YBServersRefreshInterval +
```

Comment:
The name formatting should be consistent with other property names which are following camelCase.
So we should use `ybServersRefreshInterval` at all places.